### PR TITLE
Improved support for Codeception 5/PHP 8

### PR DESCRIPTION
**Pull request by yesdevnull**
- Aligned methods with the Codeception 5 version of `Codeception\Lib\Interfaces\Web`.
- Normalised `composer.json`.

### CODE REVIEW

File: tests/data/app/db [hunk 1]
```diff
@@ -1 +1 @@
-a:0:{}
+a:4:{s:6:"params";a:0:{}s:5:"query";a:0:{}s:4:"form";a:4:{s:8:"wireless";s:5:"mouse";s:6:"coffee";s:1:"0";s:3:"tea";s:1:"1";s:7:"vanilla";s:2:"on";}s:5:"files";a:0:{}}
```
Comment (TavoNiievez) [inline tests/data/app/db:1]: Could it be that these changes have been committed by mistake?

Comment (yesdevnull) [reply tests/data/app/db:1]: It came up as a changed file after the tests ran so I presumed it was an older version of the file. I can revert if this file is meant to be an empty serialised array.

Comment (TavoNiievez) [reply tests/data/app/db:1]: It's supposed to be, revert the changes please

Comment (yesdevnull) [reply tests/data/app/db]: Done, commit dropped. Thanks!

**Comment by TavoNiievez** [comment]
@yesdevnull Thank you!

**Pull request merged**